contracts/PetPark.sol

```diff
@@ -4,4 +4,59 @@ pragma solidity ^0.8.0;
 
 contract PetPark {
 
-}
\ No newline at end of file
+    address owner;
+
+    enum Gender {Male, Female}
+    enum AnimalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+
+    struct Person {
+        uint Age;
+        Gender gender;
+        }
+
+    mapping (uint => uint) CountByAnimalType;
```

Comment:
Since animal type is an enum here (AnimalType) this should be defined as follows - `mapping (AnimalType => uint) CountByAnimalType;`

Reply:
ahh makes sense. I probably should have just stuck with int. After our gas cost optimization lecture, I am wondering if I will ever even want to use enum in solidity - I imagine it's more expensive than int.